subprojects/wrapper/src/integTest/groovy/org/gradle/integtests/WrapperGenerationIntegrationTest.groovy

```diff
@@ -71,7 +71,7 @@ class WrapperGenerationIntegrationTest extends AbstractIntegrationSpec {
         executer.inDirectory(file("second")).withTasks("wrapper").run()
 
         then: "the checksum should be constant (unless there are code changes)"
-        Hashing.sha256().hashFile(file("first/gradle/wrapper/gradle-wrapper.jar")) == HashCode.fromString("c5a643cf80162e665cc228f7b16f343fef868e47d3a4836f62e18b7e17ac018a")
+        Hashing.sha256().hashFile(file("first/gradle/wrapper/gradle-wrapper.jar")) == HashCode.fromString("577b2de036000db2e0f04f2ec842a4f1e648c8b6f9c87f29a8d896acb1732538")
```

Comment:
Alright, seems like the change is caused by https://github.com/gradle/gradle/pull/19570. I suppose we can go forward with upgrading the wrapper.

Reply:
Why does the version of Gradle we use to build Gradle affect this?

Reply:
Because the new version of Gradle is used to build the jar we put in the distribution.

Reply:
Ah, I understand.  The wrapper jar is built like any other jar. I was thinking it was built by the Gradle-under-test in this case.